tests/plugins/widget/manual/selectallsourcemode.md

```diff
@@ -0,0 +1,36 @@
+@bender-tags: 4.13.1, bug, widget, 3704
+@bender-ui: collapsed
+@bender-ckeditor-plugins: image2, wysiwygarea, toolbar, sourcearea, undo, selectall
+
+1. Open console.
```

Comment:
Should we ignore this test on mobiles since it requires opening dev console (it's doable but cumbersome and we usually tend to not do it)?